s/start.go

```diff
@@ -0,0 +1,132 @@
+package s
+
```

Comment:
I do not necessarily suggest this, but I see that this file has all private functions. Would it make sense to use an internal package? Honestly, I am not sure I like the whole "internal" thing, but it is a go'ism, so I just mention it.

Reply:
I don't think so, I think it makes sense with the `c` package, as I'm not intending for that API to be used outside of the `c.ChildSpec` opaque type and construction options. The catch is, I want to keep the `c` public APIs private to the capataz package, so `internals` sounds more reasonable there.